uportal-war/src/main/java/org/apereo/portal/portlets/backgroundpreference/RoleBasedBackgroundSetSelectionStrategy.java

```diff
@@ -78,6 +78,10 @@ public String getSelectedBackgroundImagePreferenceName() {
         public String getBackgroundContainerSelectorPreferenceName() {
             return prefix + "BackgroundContainerSelector";
         }
+
+        public String getImageCaptionsName() {
+            return "backgroundImageCaptions";
```

Comment:
Realistically, is this value ever going to change, be overridden in a sub-class, etc? Would it be better to factor it as a public static field, maybe with some JavaDoc on that field documenting it?